src/main/java/com/cloudbees/hudson/plugins/folder/ChildNameGenerator.java

```diff
@@ -293,7 +294,7 @@ public final String readItemName(@NonNull File directory) {
         File nameFile = new File(directory, CHILD_NAME_FILE);
         if (nameFile.isFile()) {
             try {
-                childName = StringUtils.defaultString(StringUtils.trimToNull(Files.readString(nameFile.toPath(), StandardCharsets.UTF_8)), directory.getName());
+                childName = Objects.toString(StringUtils.trimToNull(Files.readString(nameFile.toPath(), StandardCharsets.UTF_8)), directory.getName());
```

Comment:
I really doubt we need https://github.com/apache/commons-lang/blob/7302e9a308f0a5d59f17f72b6f5326e76cd71e3a/src/main/java/org/apache/commons/lang3/StringUtils.java#L8834-L8837 here. Not even clear to me why the original code was call `trimToNull` at all. It certainly cannot be null, it should not need to be trimmed, and it should not have been empty. If the file exists, we should load its contents as is.

Reply:
(And #483 deletes this code anyway.)

Reply:
Replaced this with the Util.fixEmptyAndTrim from core for now.

Reply:
Thanks!